### ISSUE TITLE
Add Native Return Types

### ISSUE THREAD

**Issue by seb-jean** [symfony/symfony#44366]
### Symfony version(s) affected

6.0

### Description

Hi,
I'm on Windows.
I made the command "vendor\bin\patch-type-declarations.bat" but I have errors:

```
Fatal error: Uncaught Error: Class "Doctrine\DBAL\Tools\Console\Command\ImportCommand" not found in C:\wamp64\www\my_project_name\vendor\doctrine\doctrine-bundle\Command\Proxy\ImportDoctrineCommand.php:17
Stack trace:
#0 C:\wamp64\www\my_project_name\vendor\symfony\error-handler\DebugClassLoader.php(287): include()
#1 [internal function]: Symfony\Component\ErrorHandler\DebugClassLoader->loadClass('Doctrine\\Bundle...')
#2 C:\wamp64\www\my_project_name\vendor\symfony\error-handler\Resources\bin\patch-type-declarations(82): class_exists('Doctrine\\Bundle...')
#3 {main}
  thrown in C:\wamp64\www\my_project_name\vendor\doctrine\doctrine-bundle\Command\Proxy\ImportDoctrineCommand.php on line 17
```

### How to reproduce

Type  command "vendor\bin\patch-type-declarations.bat"

**Comment by nicolas-grekas** [comment]
Please provide a reproducer that someone else could locally. Alternatively, can you figure out the issue?

**Comment by YetiCGN** [comment]
Please check if you have `doctrine/dbal` 3.x. The ImportCommand had been deprecated for a long time now and has been removed. Explicitly requiring DBAL ^2.13.6 should bring it back for now.

**Comment by stof** [comment]
@nicolas-grekas does the patching script handle properly classes that have an undefined parent class ? DoctrineBundle registers is proxy ImportCommand conditionally based on the existence of the DBAL class, but loading the file unconditionally in the script would produce a broken inheritance chain.

**Comment by seb-jean** [comment]
> Please check if you have `doctrine/dbal` 3.x. The ImportCommand had been deprecated for a long time now and has been removed. Explicitly requiring DBAL ^2.13.6 should bring it back for now.

I have `doctrine/dbal` 3.2.0

My project is a project initialized yesterday via the command:
`symfony new my_project_name --full`

**Comment by xabbuh** [comment]
Can you create a small example application that allows to reproduce your issue?

**Comment by seb-jean** [comment]
Here is a small example application that allows to reproduce issue:
https://github.com/seb-jean/my_project_name_sf

**Comment by nicolas-grekas** [comment]
Thanks for the reproducer. Fixed in #44719
